release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -1860,6 +1872,7 @@ Workaround: Add the mandatory `spec.git_provider.user` field for Bitbucket. In a
 
 * Currently, a certificate compliance issue, specifically outputted as `x509: certificate is not standards compliant`, exists when you run the `openshift-installer` on macOS for the purposes of installing an {product-title} cluster on VMware vSphere. This issue relates to a known issue with the `golang` compiler in that the compiler does not recognize newly supported macOS certificate standards. No workaround exists for this issue. (link:https://issues.redhat.com/browse/OSDOCS-5694[*OSDOCS-5694*])
 
+* The load balancing algorithm did not prioritize existing control plane machines when more than three failure domains were included in the `ControlPlaneMachineSet` definition. If you added a fourth failure domain that was alphabetically higher in precedence than the existing three failure domains to the definition, this fourth failure domain take precedence over any existing failure domains. This behavior would cause rolling forward updates to be applied to a control plane machine. To avoid this scenario, ensure that existing in-use failure domains retain a higher precedence than the new and unused failure domains. By completing this action, each control plane machine remains stable during the course of adding more than three failure domains to the definition. (link:https://issues.redhat.com/browse/OCPBUGS-11968[*OCPBUGS-11968*])
```

Comment:
```suggestion
* When more than three failure domains are included in the `ControlPlaneMachineSet` definition, the load balancing algorithm does not prioritize existing control plane machines. If you add a fourth failure domain that is alphabetically higher in precedence than the existing three failure domains to the definition, the fourth failure domain takes precedence over any existing failure domains. This behavior causes rolling forward updates to be applied to a control plane machine. As a workaround, set existing in-use failure domains to a higher precedence than the new and unused failure domains. This workaround stabilizes control plane machines during the course of adding more than three failure domains to the definition. (link:https://issues.redhat.com/browse/OCPBUGS-11968[*OCPBUGS-11968*])
```

Reply:
Present tense and focused on actions the user can take.

Reply:
Removed past tense and passive voice. 

"Workaround" is classed as jargon, so I did not implement this change:
![Screenshot from 2023-04-28 10-36-57](https://user-images.githubusercontent.com/57954076/235112953-50859e96-636e-48dc-98dd-3d41b18b09ce.png)